### PR TITLE
Fix/add banners to stdout with junit

### PR DESCRIPTION
**Pull request by sebclrsn**
Console output is now 100% identical (and XML report hasn't changed)

### CODE REVIEW

File: src/pykiso/test_result/text_result.py [hunk 1]
```diff
@@ -19,17 +19,25 @@
...
-from unittest import TestCase, TestResult, TextTestResult
+from types import TracebackType
+from typing import List, Optional, TextIO, Tuple, Type, Union
+from unittest import TextTestResult
+
+from ..test_coordinator.test_case import BasicTest
+from ..test_coordinator.test_suite import BaseTestSuite
 
-from pykiso.types import PathType
+if typing.TYPE_CHECKING:
```
[the remark this answers is not in the document]
Comment (BKaDamien) [reply src/pykiso/test_result/text_result.py:39]: circular import?

File: src/pykiso/test_coordinator/test_case.py [hunk 2]
```diff
@@ -227,7 +156,8 @@ def __init__(
             wait for a report during teardown execution
         :param test_ids: jama references to get the coverage
             eg: {"Component1": ["Req1", "Req2"], "Component2": ["Req3"]}
-        :param tag: dictionary containing lists of variants and/or test levels when only a subset of tests needs to be executed
+        :param tag: dictionary containing lists of variants and/or test
```
Comment (sebastianpfischer) [inline src/pykiso/test_coordinator/test_case.py:159]: Should be changed to something more generic:
```suggestion
        :param tag: Dictionary allowing users to filter the tests based on the keys and their value.
```

**Review by Pog3k — APPROVED** [review]

**Review by sebastianpfischer — APPROVED** [review]

File: src/pykiso/test_coordinator/test_case.py [hunk 3]
```diff
@@ -123,8 +51,8 @@ def __init__(
         teardown_timeout: Union[int, None],
         test_ids: Union[dict, None],
         tag: Union[Dict[str, List[str]], None],
-        args: tuple,
-        kwargs: dict,
+        args: Any,
```
[the remark this answers is not in the document]
Comment (sebclrsn) [reply src/pykiso/test_coordinator/test_case.py:54]: No idea why this was defined like this at the beginning, maybe you know?
But I agree with the typing, as there they aren't packed tuple and dict are the correct types (looked too fast there)

File: src/pykiso/test_result/xml_result.py [hunk 4]
```diff
@@ -126,5 +157,5 @@ def _report_testcase(
         # here can be added additional tags that have to be stored into the xml test report
         xml_testsuite.setAttribute("test_ids", str(test_result.test_ids))
 
-    # Catch and redefine xmlrunner.runner._XMLTestResult._report_testcase
+    # Overwrite the original staticmethod xmlrunner.runner._XMLTestResult._report_testcase with ours
```
[the remark this answers is not in the document]
Comment (sebclrsn) [reply src/pykiso/test_result/xml_result.py:160]: Don't know if overriding applies to a static methods also but sure I'll update the comment

**Review by BKaDamien — APPROVED** [review]

File: src/pykiso/test_result/text_result.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebclrsn) [reply src/pykiso/test_result/text_result.py:39]: In that case it isn't required, but I find it always nice to have the dependencies that are only required for typing in there, like that you can directly see what's concretely used in the code and what's only imported for annotations

Should I revert this?

**Pull request merged**